tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -26,6 +26,15 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
   var validGroupId: GroupId = null
   var validDocumentId: DocumentId = null
 
```

Comment:
We should have one or more tests that timeout on functions that do not pass a timeout explicitly (use the timeout set at initialize time), just to make sure that's all working properly.

Reply:
I don't know if I can easily show this. If I pass a `config` with a short `timeout` like 5ms, the `initialize` itself will fail, so I can't try any sdk functions with it.

Reply:
Yep. You're right. That's the same thing I found in ironoxide, actually :P